src/softwareHeritage/agent/softwareHeritageAgent.php

```diff
@@ -124,6 +124,7 @@ function processEachPfileForSWH($pfileDetail, $agentId, $maxTime)
     if (SoftwareHeritageDao::SWH_RATELIMIT_EXCEED == $currentStatus) {
       $this->heartbeat(0); //Fake heartbeat to keep the agent alive.
       $timeToReset = $currentResult - time();
+      print "INFO :Software Heritage X-RateLimit-Limit reached. Next slot unlocks in ".gmdate("H:i:s", $timeToReset)." ";
```

Comment:
```suggestion
      print "INFO :Software Heritage X-RateLimit-Limit reached. Next slot unlocks in ".gmdate("H:i:s", $timeToReset)."\n";
```

While testing the PR, I saw some unusual logs:

```
"INFO :Software Heritage X-RateLimit-Limit reached. Next slot unlocks in 00:56:05 HEART: 47 1"
"INFO :Software Heritage X-RateLimit-Limit reached. Next slot unlocks in 00:55:40 HEART: 47 1"
"INFO :Software Heritage X-RateLimit-Limit reached. Next slot unlocks in 00:55:09 HEART: 47 1"
"INFO :Software Heritage X-RateLimit-Limit reached. Next slot unlocks in 00:54:37 HEART: 47 1"
```

Potentially the hearbeats to the scheduler are getting lost. Adding the newline to the end of message improved the situation:

```
"INFO :Software Heritage X-RateLimit-Limit reached. Next slot unlocks in 00:50:22"
"INFO :Software Heritage X-RateLimit-Limit reached. Next slot unlocks in 00:49:53"
"INFO :Software Heritage X-RateLimit-Limit reached. Next slot unlocks in 00:49:21"
"INFO :Software Heritage X-RateLimit-Limit reached. Next slot unlocks in 00:48:50"
```

Please have a look @shaheemazmalmmd